### ISSUE TITLE
unexpected workingDiff return value for days unit

### ISSUE THREAD

**Issue by Guria** [lennym/moment-business-time#8]
[readme](https://github.com/lennym/moment-business-time#momentworkingdiff):

> Calculates the difference between two moments, counting only working time. Arguments are as per moment#diff

[moment.js docs](http://momentjs.com/docs/#/displaying/difference/):

> If the moment is earlier than the moment you are passing to moment.fn.diff, the return value will be negative.
> A easy way to think of this is by replacing .diff( with a minus operator.
> 
> ``` js
> // a < b
> a.diff(b) // a - b < 0
> b.diff(a) // b - a > 0
> ```

actual:

``` js
// a < b
var a = moment().startOf('week')
var b = moment().endOf('week')
// ok with 'hours'
a.workingDiff(b, 'hours') // -40
b.workingDiff(a, 'hours') // 40
// wtf with 'days'
b.workingDiff(a, 'days')  // -5
a.workingDiff(b, 'days')  // 5
```

**Comment by lennym** [comment]
Thanks for raising an issue @Guria, I'll take a look at it.

**Comment by lennym** [comment]
Confirmed this is an actual bug in the code. I'm working on a fix now.

**Comment by lennym** [comment]
Published at 0.4.0

Thanks for spotting this again @Guria

**Comment by Guria** [comment]
:+1:
